File: python/Resepsionis.py
class Resepsionis:

    def __init__(self, tiket=0, tanggal="", studio=0, film=""):
        self.__tiket = tiket
        self.__tanggal = tanggal
        self.__studio = studio
        self.__film = film

    def setTiket(self, tiket):
        self.__tiket = tiket

    def getTiket(self):
        return self.__tiket

    def setTanggal(self, tanggal):
        self.__tanggal = tanggal

    def getTanggal(self):
        return self.__tanggal

    def setStudio(self, studio):
        self.__studio = studio

    def getStudio(self):
        return self.__studio

    def setFilm(self, film):
        self.__film = film

    def getFilm(self):
        return self.__film

File: java/Resepsionis.java

class Resepsionis {
    
    private int tiket;
    private String tanggal;
    private int studio;
    private String film;

    Resepsionis(){
    }

    Resepsionis(int tiket, String tanggal, int studio, String film) {
        this.tiket = tiket;
        this.tanggal = tanggal;
        this.studio = studio;
        this.film = film;
    }
    

    public void setTiket(int tiket){
        this.tiket = tiket;
    }

    public int getTiket(){
        return this.tiket;
    }

    public void setTanggal(String tanggal){
        this.tanggal = tanggal;
    }

    public String getTanggal(){
        return this.tanggal;
    }

    public void setStudio(int studio){
        this.studio = studio;
    }

    public int getStudio(){
        return this.studio;
    }

    public void setFilm(String film){
        this.film = film;
    }

    public String getFilm(){
        return this.film;
    }

}
